### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yagneshprajapati**
feature section in table format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This repository is a merger of the formerly separate GoogleTest and GoogleMock
 projects. These were so closely related that it makes sense to maintain and
 release them together.
 
-### Getting Started
+### | Getting Started |
```
Comment (derekmauro) [inline README.md:44]: Why are there | chars around Getting Started?

File: README.md [hunk 2]
```diff
@@ -50,18 +50,19 @@ documentation. We recommend starting with the
...
-*   User-defined assertions.
-*   Death tests.
-*   Fatal and non-fatal failures.
-*   Value-parameterized tests.
-*   Type-parameterized tests.
-*   Various options for running the tests.
-*   XML test report generation.
+| Feature | Description |
+| --- | --- |
+| xUnit test framework | Googletest is based on the xUnit testing framework, a popular architecture for unit testing |
```
Comment (derekmauro) [inline README.md:55]: Can you restore the xUnit link?

**Review by derekmauro — CHANGES_REQUESTED** [review]
Can you update the description of this change to describe what is changing? "Update README.md" is too general a description. Please describe how it is changing.

File: README.md (hunk 1 above)
Comment (yagneshprajapati) [reply README.md:44]: it indicate that "Getting Started" is a heading in a larger body of text, also drawing attention to it.

File: README.md (hunk 2 above)
Comment (yagneshprajapati) [reply README.md]: yes done!!

**Comment by yagneshprajapati** [comment]
now looks good

File: README.md (hunk 1 above)
Comment (derekmauro) [reply README.md]: Please take a look at the preview of the file: https://github.com/google/googletest/blob/a7fa9b37082993107a98181e631cafcdbe484f22/README.md#-getting-started-

I don't think the markdown is rendering as you intend.

**Comment by yagneshprajapati** [comment]
> See my comment on the rendering.

yes agree about the | char is may be not well. so my recent commit is ok?

**Pull request merged**